oamapps/postConfigure/mycnfUpgrade.cpp

```diff
@@ -144,7 +201,7 @@ int main(int argc, char *argv[])
 					ofstream newFile (mycnfFile.c_str());	
 					
 					//create new file
-					int fd = open(mycnfFile.c_str(), O_RDWR|O_CREAT, 0666);
+					int fd = open(mycnfFile.c_str(), O_RDWR|O_CREAT, 0644);
```

Comment:
I'll take this out.  Fixing our file permissions is a whole Thing, and outside of the scope of this ticket.